.mergify.yml

```
@@ -4,7 +4,7 @@ pull_request_rules:
       - check-success=ci/hercules/evaluation
       - check-success=ci/hercules/derivations
       - author=github-actions[bot]
-      - head~=^update/(nixpkgs|sops-nix|hercules-ci-effects,cachix)-
+      - head~=^update/(nixpkgs|sops-nix|hercules-ci-effects|cachix)-
```

Could remove it as well I guess?

```suggestion
      - head~=^update/(nixpkgs|sops-nix|hercules-ci-effects)-
```